### DISCUSSION TITLE
Pull InfluxDB data into another Grafana instance?

### DISCUSSION THREAD

**Discussion by sjnhawk** [Security-Onion-Solutions/securityonion#9815] [Unsupported Versions]
Is it possible to pull Security Onion data from InfluxDB into another Grafana instance?

I have Grafana running on another server that aggregates information from other systems/servers on my network. I have some dashboards configured to show general health information (CPU, RAM, Swap, etc.) and I was hoping to pull that information from Security Onion as well so that I don't have to switch between Grafana instances. Hope that makes sense?

I tried to add Security Onion's InfluxDB as an additional data source to my local Grafana by mirroring the Data Sources/Influx DB on Security Onion, but get the following when I save and test...

"Get "https://X.X.X.X:8086/query?db=telegraf&epoch=ms&q=SHOW+measurements": dial tcp X.X.X.X:8086: i/o timeout error performing influxQL query"

I was hoping that I could just tap into it without creating any API keys or making config changes to Security Onion's InfluxDB.

**Comment by andre-dessert** [comment]
This is not something we recommend or support due to the way we use the Influx data.
